media-gfx/openscad/openscad-2021.01-r4.ebuild

```diff
@@ -78,7 +78,8 @@ src_configure() {
 }
 
 src_compile() {
-	default
+	# We have random parallel build issues, bug #856430
+	MAKEOPTS="-j1" emake
```

Comment:
Thanks! I've gone for `emake -j1` instead to not completely override MAKEOPTS.

Reply:
I tried this, but it didn't work and still used the value of MAKEOPTS, so I switched to this way.